backend/src/routes/question.js

```diff
@@ -43,4 +44,16 @@ question.put('/:id/likes', questionController.putQuestionLikes)
 /* Put an updated question's answer like value */
 question.put('/:id/answer/:answerID/likes', questionController.putAnswerLikes)
 
+/* Report a question */
+question.post('/:id/report', reportController.postReport('question'))
```

Comment:
'question', 'answer' .. etc should probs be CONSTs used elsewhere I noted.

Reply:
I'm not sure tbh. Do you mean the table names used elsewhere? For these strings, I actually use them in the controller for stuff...
The alternative I was gonna do was have postQuestionReport, postAnswerReport functions etc and have them all redirect to a helper, but I thought this was a little easiest